src/main/java/org/jitsi/xmpp/mucclient/MucClient.java

```diff
@@ -426,6 +426,16 @@ private IQ handleIq(IQ iq)
     {
         IQ responseIq = null;
 
+        EntityBareJid fromJid = iq.getFrom().asEntityBareJidIfPossible();
+        String fromJidStr = fromJid.toString().toLowerCase();
+        if (fromJid == null
+            || !this.config.getMucJids().stream().anyMatch(
+                    mucJid -> mucJid.toLowerCase().equals(fromJidStr)))
+        {
+            logger.warn("Received an IQ with unauthorized from:" + fromJid);
```

Comment:
`"Received an IQ from a non-MUC member: " + fromJid"`?

Reply:
We reject everything that is not coming from the muc, for example when someone is contacting us directly by jid.

Reply:
sorry, i was suggesting we re-word the log message

Reply:
Yep, thanks :) Fixed.